editor.planx.uk/src/pages/layout/PublicLayout.tsx

```diff
@@ -34,7 +34,6 @@ interface PublicLayoutProps {
 
 const MainContainer = styled(Box)(({ theme }) => ({
   borderTop: `1px solid ${theme.palette.grey[300]}`,
-  paddingTop: theme.spacing(5),
```

Comment:
This is hopefully covered by the associated change in `editor.planx.uk/src/pages/Preview/Questions.tsx`, but please let me know if anybody spots weird spacing anywhere.